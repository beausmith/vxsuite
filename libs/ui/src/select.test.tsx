import React from 'react';
import { render } from '@testing-library/react';

import { Select } from './select';

describe('renders Select', () => {
  test('large', () => {
    const { container } = render(<Select large />);
    expect(container.firstChild).toMatchSnapshot();
  });
  test('small', () => {
    const { container } = render(<Select small />);
    expect(container.firstChild).toMatchSnapshot();
  });
  test('fullWidth', () => {
    const { container } = render(<Select fullWidth />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
