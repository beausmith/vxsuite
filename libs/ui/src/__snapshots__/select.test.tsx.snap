// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Select fullWidth 1`] = `
<select
  class="sc-bdvvaa dNULwi"
/>
`;

exports[`renders Select large 1`] = `
<select
  class="sc-bdvvaa SskPc"
/>
`;

exports[`renders Select small 1`] = `
<select
  class="sc-bdvvaa fhIuoJ"
/>
`;
